### DISCUSSION TITLE
EventListener for SDK V3

### DISCUSSION THREAD

**Discussion by ruspamintuan** [aws/aws-sdk-js-v3#5163] [Q&A]
Hello, we're currently migrating to v3. In our previous implementation, we are using an event listener for "retry" event and do a simple logging to see if our retry function did indeed run. Here's an example from this doc:

https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/Request.html#retry-event

```
request.on('retry', function(response) {
  // this resource is not yet available, wait 10 seconds to get it again
  if (response.httpResponse.statusCode === 404 && response.error) {
    response.error.retryable = true;   // retry this error
    response.error.retryDelay = 10000; // wait 10 seconds
  }
});
```

Does anybody know how to implement this in v3? Currently have not found any doc about this. Thank you!

**Comment by Tejaswinisana** [comment]
Looking for exactly the same. Did you find anything yet ?

**Reply by ruspamintuan** [reply]
As of now, still haven't found anything
